src/Microsoft.Health.Fhir.Api/Modules/SecurityModule.cs

```diff
@@ -52,6 +52,7 @@ public void Load(IServiceCollection services)
                         options.Authority = _securityConfiguration.Authentication.Authority;
                         options.Audience = _securityConfiguration.Authentication.Audience;
                         options.RequireHttpsMetadata = true;
+                        options.Challenge = $"Bearer authorization_uri={_securityConfiguration.Authentication.Authority}, resource_id={_securityConfiguration.Authentication.Audience}, realm={_securityConfiguration.Authentication.Audience}";
```

Comment:
I think this is actually supposed to be the authorization URI, so:

```
$"Bearer authorization_uri={_securityConfiguration.Authentication.Authority}/oauth2/authorize, resource_id={_securityConfiguration.Authentication.Audience}, realm={_securityConfiguration.Authentication.Audience}"
```

Reply:
Actually, upon looking into this. Your code is correct. I had noticed some products that do it differently, but this is correct according to AAD docs.